src/Data/Quantity/Physics.purs

```diff
@@ -61,15 +61,15 @@ electronCharge = 1.60217653e-19 .* coulomb
        Right q → q
        Left _ → ε0'
   where
-    ε0' = scalar 1.0 ⊘ (µ0 ⊗ (speedOfLight `pow` (fromNumber 2.0)))
+    ε0' = scalar 1.0 ⊘ (µ0 ⊗ (speedOfLight `pow` (fromInt 2)))
```

Comment:
I don't think we gain anything from this change and similar ones below. Do you think I should revert them?

Reply:
No, I like it - thanks!